pkg/loki/loki.go

```diff
@@ -134,6 +134,30 @@ func (c *Config) registerServerFlagsWithChangedDefaultValues(fs *flag.FlagSet) {
 	})
 }
 
+func (c *Config) registerQueryRangeFlagsWithChangedDefaultValues(fs *flag.FlagSet) {
+	throwaway := flag.NewFlagSet("throwaway", flag.PanicOnError)
+
```

Comment:
```suggestion
	// NB: We can remove this after removing Loki's dependency on Cortex and bringing in the queryrange.Config.
	// That will let us change the defaults there rather than include wrapper functions like this one.
```